fe/fe-core/src/main/java/com/starrocks/lake/vacuum/AutovacuumDaemon.java

```diff
@@ -208,10 +239,16 @@ private void vacuumPartitionImpl(Database db, OlapTable table, PhysicalPartition
         }
 
         partition.setLastVacuumTime(startTime);
+        if (!hasError && vacuumedVersion > partition.getLastSuccVacuumVersion()) {
+            // hasError is false means that the vacuum operation on all tablets was successful.
+            // the vacuumedVersion isthe minimum success vacuum version among all tablets within the partition which
+            // means that all the garbage files before the vacuumVersion have been deleted.
+            partition.setLastSuccVacuumVersion(vacuumedVersion);
```

Comment:
is it possible that the partition is an empty partition without tablets at the beginning, and cause this vacuum to LONG_MAX and later on some tablets are added into the partition, and the vacuumed version will be never reset to the correct one?

Reply:
> is it possible that the partition is an empty partition without tablets at the beginning, and cause this vacuum to LONG_MAX and later on some tablets are added into the partition, and the vacuumed version will be never reset to the correct one?

I think this should not happen. Because an empty partition won't perform the vacuum operation.